dags/vz_cr3_download.py

```diff
@@ -0,0 +1,102 @@
+import os
+
+from airflow.decorators import task
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+from airflow.operators.dummy_operator import DummyOperator
+from pendulum import datetime, duration, now, parse
+
+from utils.onepassword import get_env_vars_task, get_item_last_update_date
+from utils.slack_operator import task_fail_slack_alert, task_success_slack_alert
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+
+default_args = {
+    "owner": "airflow",
+    "description": "Downloads CR3 pdfs from CRIS and uploads to S3",
```

Comment:
i've had a hard time finding the docs on this, but i'm pretty sure you want to pass this property to the `DAG()` constructor instead. we have a bad pattern in our v1 DAGs.